Assets/UI Toolkit/Scripts/ExtensionMethods/VisualElementExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Text;
+using UnityEngine.UIElements;
+
+namespace Netherlands3D.UI.ExtensionMethods
+{
+    public static class VisualElementExtensions
+    {
+        public static void SetFieldValueAndReplaceClassName<T>(
```

Comment:
Supporting method to remove the previous class name if one was set, and set a new one. This saves a ton of boilerplate code